### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by Mucyosoda**
the project capstone is building in HTML, CSS, and javascript, project has two pages that are the home page and the about page
Javascript is used to generate cards for featured speakers, cards information generate dynamically when the page reloads. This project has a mobile version and a desktop version.

- Link video that describes my project: https://www.loom.com/share/f1bc58481f024f8186968522df90a809

### CODE REVIEW

**Comment by mikenath223** [comment]
Hi , 

You made a great effort in this project, but unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that will be invalidated for this reason:
-  Kindly remove node_module from the list of git tracked files and include it in gitignore.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Happy coding. Cheers!🚀🚀

If something isn't 100% clear, you can leave comments or questions in this PR thread.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **don't count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            </li>
+            <li><a href="#">English</a>
+                <li><a href="#">My Page</a>
+                    <li><a href="#">Logout</a>
+        </ul>
+    </div>
+    <header id="header">
+        <div class="logo">
+          <a href="index.html">
+          <img src="images/gorilla-logo.png" alt=""></a>
```
Comment (rloterh) [inline index.html]: - [ ] OPTIONAL
It's a good practice to provide `alt` names for your images so as to improve the accessibility features of your site.  Kindly endeavor to add `alt` names to **all** your images.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+    subtitle: 'His president',
+    text: 'He served as chair of AU from 2018',
+    image: {
+      src: 'images/b7.jpg',
+      alt: 'president',
+    },
+  },
+];
+
+// creating card one
```
Comment (rloterh) [inline script.js]: - [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li><a href="#">English</a>
+                <li><a href="#">My Page</a>
+                    <li><a href="#">Logout</a>
+        </ul>
+    </div>
```
Comment (rloterh) [inline index.html]: - [ ] OPTIONAL
Your code is not properly indented. Kindly ensure that your code has consistent **spacing** for indentation. The recommended spacing is **2 spaces**. You can install and configure a code formatter plugins such as [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) in your code editor for formatting your code to ensure consistent and well-indented code.

File: aboutpage.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+            <a class="nav-link" href="#" >Join</a>
+            <a class="nav-link" href="#" >News</a>
+            <a class="nav-link" href="#" >Sponsor</a>
+            <a class="nav-link" id="home" href="index.html">RW Compaign</a>
+          </div>
+        </nav>
+        <div class="menu">
+          <i class="fas fa-bars"></i>
+        </div>
+      </header>
```
Comment (rloterh) [inline aboutpage.html]: The top dark navbar and the bottom white navbar of your desktop version do not match the provided [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly note we actually want our design to as much as possible _mimic_ the design of the provided template design. Kindly ensure to improve upon these aspects:
- [ ] The **height** of the top dark navbar is **shorter**, as can be seen in the original design below.
- [ ] The content of the dark navbar is **vertically centered** and **left-aligned** with the contents of the white navbar.
- [ ] The **typography** (font family, font-weight and font size, and the like ) **should match** the design of the provided template.
- [ ] The contents on both navbars are **centered** and that that the _padding_ of the extreme left and rights sides of the nav contents are equal


**Your header navigation sections:**
image

<br>

**The original header navigation sections:**
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+             <div class="text_content">
+                 <h2>Rwanda</h2>
+                 <h4>His President of the Republic of Rwanda. </h4>
+                 <h5>
+                    He served as Chair of the AU from 2018 to 2019 and chaired the East African Community from 2018-2021.
+                 </h5>
+             </div>
+            </div>
+        </div>
+    </div>
```
Comment (rloterh) [inline index.html]: - [ ] Typography (**font family**, **font-weight**, **font size**, **vertical height**, **font-style**...) and **spacing** of contents your _feature speakers_ section does not match the provided [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).  Kindly ensure to improve upon this aspects

**Your feature speakers section:**
image


**The original feature speakers section:**
image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+          <div class="logoname">Gorila Creative</div>
+            <div class="footer_text">
+                <h3>2021 Creative Commons Rwanda. Some Rights Reserved.</h3>
+                <p>
+                    His Excellency Paul Kagame, President of Rwanda, and Commonwealth Secretary-General, The Rt. <br> Hon. Patricia Scotland QC, are today announcing the postponement of CHOGM 2021 as a result of <br> the continuing impact of the Covid-19 pandemic.  
+                </p>
+
+            </div>
+        </div>
+    </footer>
```
Comment (rloterh) [inline index.html:209]: The **background color** of your footer on both the mobile and desktop versions are the same. As specified let's ensure to improve upon these aspects:
- [x]  The footer, on mobile size should have a white background, which is implemented.
- [ ] The footer, on desktop size should have a dark background.

File: README.md [hunk 7]
```diff
@@ -1,75 +1,44 @@
...
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- Javascript
+- HTML
+- CSS
```
Comment (rloterh) [inline README.md:16]: - [ ] Your Pull Request does not include a video presentation showcasing your project. Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project. The video should:
- contain a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
- contain a demo of the project, showing the pages in different screen sizes.
- contain a highlight about a piece of the code the student finds interesting or is proud of making.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+            <i class="fas fa-times-circle"></i>
+          </div>
+      </header>
+    <div class="banner" >
+        <div class="background">          
+        <h1>"Rwanda Film Festival"</h1>
+        <h2>Visit Rwanda Festivals <br> and Events</h2>
+        <p>If you’re looking to get out of the big city and back into nature, the annual Kwita Izina gorilla-naming ceremony is a special time to visit Volcanoes National Park. The annual naming ceremony takes place in Kinigi,</p>
+        <h3>24th SEPTEMBER 2021 SAVE THE DATE.</h3>
+    </div>
```
Comment (rloterh) [inline index.html]: - [ ] The typography(font family, font-weight, font size,...), headings texts, paragraphs, and spacing contents of your _hero_/ _banner_ section do not match the provided [desgn template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Once again, we actually want our design to as much as possible closely mimic the design of the provided template. The Home page should contain a _top_ section with a column that includes a **small line of text**, **the page title**, a **block of text**, a **medium line of text in bold letters**, and a **smaller line of text**, as can be seen in the provided template design. Let's ensure that the **font family**, font sizes, **font weights**, **font colors**, and **letter-spacing** matches that of the provided template design

**Your hero design:**
image


**The original hero design:**
image

File: index.html [hunk 9]
```diff
@@ -0,0 +1,177 @@
...
+    </div>
+    <main id="programs">
```
Comment (rloterh) [inline index.html:91]: - [ ] The items within the _main program_ section of your design do not follow the typography (**font family**, **font-weight**, ...), and do not have appropriate **padding** and **margins** around the items, as well as **spacing** between the contents of each item. Kindly ensure that there's enough padding around the item's contents.

**Your program section:**
image

**The original program section:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mucyo, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Good video presentation
- Nice giving acknowledgment to **God**, and  Cindy Shin the author of the original design,
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 9 above)
Comment (emmanuellekamwa) [reply index.html:91]: - [ ] As said by the previous code reviewer, try as much as possible to respect the original design of the project. The spacing between the icons and the titles is too much and the font family is not respected. Please kindly fix that.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑🛠**
Hello @Mucyosoda 👋🏽
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._👀👇🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+        <p>
+          If you’re looking to get out of the big city and back into nature, the
+          annual Kwita Izina gorilla-naming ceremony is a special time to visit
+          Volcanoes National Park. The annual naming ceremony takes place in
+          Kinigi,
+        </p>
+        <h3>24th SEPTEMBER 2021 SAVE THE DATE.</h3>
+      </div>
+    </div>
+    <main id="programs">
```
Comment (rloterh) [inline index.html:91]: This issue is not properly resolved.
- [ ] Your _main program_ section does not follow the typography (**font family**, **font-weight**, ...). Kindly ensure to use the appropriate fonts properties (**font family**, **font-weight**, font size...)). Please bear in mind that we want our design to as much as possible follow the provided design template as much as possible.

**Your program section:**
image


**The original program section:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mucyo, 

You did great on this project, and for making the changes requested by the previous reviewer! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below, made under the reviews of the previous code reviewer(s) and stating **not properly resolved**. Kindly take your time to correctly review those issues outlined by the previous code reviewers, and please check to see that the changes are working as intended._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 10 above)
Comment (Mucyosoda) [reply index.html:91]: this should be the browser zoom size that might be different, mine is 100% zoom

**Review by george-shammar — APPROVED** [review]
Hi @Mucyosoda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**